modules/lvms-troubleshooting-recovering-from-missing-lvms-or-operator-components.adoc

```diff
@@ -67,9 +48,14 @@ topolvm-node-dr26h                    4/4     Running   0             66m
 vg-manager-r6zdv                      1/1     Running   0             66m
 ----
 +
-The expected output is one running instance of `lvms-operator` and `vg-manager`. One instance of `topolvm-controller` and `topolvm-node` is expected for each node.
+The output of this command must contain a running instance of the following pods:
+
+* `lvms-operator`
+* `vg-manager`
+* `topolvm-controller`
+* `topolvm-node`
 +
-If `topolvm-node` is stuck in the `Init` state, there is a failure to locate an available disk for LVMS to use. To retrieve the information necessary to troubleshoot, review the logs of the `vg-manager` pod by running the following command:
+If the `topolvm-node` pod is stuck in the `Init` state, it is due to a failure to locate an available disk for {lvms} to use. To retrieve the necessary information to troubleshoot this issue, review the logs of the `vg-manager` pod by running the following command:
```

Comment:
```suggestion
If the `vg-manager` pod is stuck trying to load a configuration file, it is due to a failure to locate an available disk for {lvms} to use. To retrieve the necessary information to troubleshoot this issue, review the logs of the `vg-manager` pod by running the following command:
```

This needs to change to check for vg-manager being stuck for 4.16+

Reply:
As there are two updates specific to 4.16+, I will make note of these and create another PR to make these updates only in 4.16.